docs/using.md

```diff
@@ -400,11 +413,27 @@ For example, to export the results of the Insert operation in the InsertRemove w
 ./run-genny export build/WorkloadOutput/CedarMetrics/InsertRemoveTest.Insert.ftdc -o insert.csv
 ```
 
-You can also use the `translate` subcommand to convert results to a [t2-readable](https://github.com/10gen/t2/) format.
+You can also use the `translate` subcommand to convert results to a
+[t2-readable](https://github.com/10gen/t2/) format.
 
-If you are running Genny through DSI in Evergreen, the FTDC contents are rolled up into summary statistics like `OperationThroughput` and such, viewable in the Evergreen perf UI. 
+If you are running Genny through DSI in Evergreen, the FTDC contents are rolled up into summary
+statistics like `OperationThroughput` and such, viewable in the Evergreen perf UI. 
 
-If you are running Genny locally, you can use `src/workloads/contrib/analysis/test_result_summary.py` to print a summary of the most recent run (or any `CedarMetrics` directory) to the console. For example,
+<a id="org2f7c9d4)"></a>
+
+### Analyzing workload output locally
```

Comment:
This is where the main part of the change is.

Reply:
Thanks for adding this, sorely needed.